pipelines/controller/src/main/java/com/google/fhir/analytics/DataProperties.java

```diff
@@ -179,6 +176,9 @@ PipelineConfig createBatchOptions() {
     }
     options.setViewDefinitionsDir(Strings.nullToEmpty(viewDefinitionsDir));
     options.setSinkDbConfigPath(Strings.nullToEmpty(sinkDbConfigPath));
+    if (rowGroupSizeForParquetFiles > 0) {
+      options.setRowGroupSizeForParquetFiles(rowGroupSizeForParquetFiles);
```

Comment:
Question: In the previous PR where you added the row-group size option to the controller, did you check the memory usage of the parquet-merger step when triggered by the controller? Or did you only check that by running the merger directly from the command line?